tests/x11/firefox/firefox_appearance.pm

```diff
@@ -31,6 +31,9 @@ sub run {
     type_string "addons.mozilla.org/en-US/firefox/addon/opensuse\n";
     assert_screen('firefox-appearance-mozilla_addons', 90);
     assert_and_click "firefox-appearance-addto";
+    if (check_screen("firefox-appearance-addto-permissions_requested", 10)) {
+        assert_and_click "firefox-appearance-addto-allow";
+    }
```

Comment:
Either follow @dzedro's idea or please use a multi-tag assert_screen with match_has_tag, e.g.

```perl
assert_screen([qw(firefox-appearance-installed firefox-appearance-addto-permissions_requested)]);
assert_and_clilck 'firefox-appearance-addto-allow' if match_has_tag 'firefox-appearance-addto-permissions_requested';

Reply:
@okurz, may I ask why this is a preferred way? As I still need to assert if it is installed, the code will be more complicated (and I do not like the duplication of the assertion).
```
assert_screen([qw(firefox-appearance-installed firefox-appearance-addto-permissions_requested)], 90);
if (match_has_tag("firefox-appearance-addto-permissions_requested")) {
    assert_and_click "firefox-appearance-addto-allow";
    assert_screen('firefox-appearance-installed', 90);
}
```